e2e/cypress/tests/integration/enterprise/bot_accounts/managing_bot_accounts_spec.ts

```diff
@@ -10,26 +10,33 @@
 // Stage: @prod
 // Group: @enterprise @bot_accounts
 
+import {AdminConfig} from '@mattermost/types/lib/config';
+
 import * as TIMEOUTS from '../../../fixtures/timeouts';
 
 describe('Managing bot accounts', () => {
-    let botName;
+    let botName: string;
+    let origConfig: AdminConfig;
 
     beforeEach(() => {
         cy.apiRequireLicenseForFeature('LDAP');
 
         cy.apiAdminLogin();
 
-        // # Set ServiceSettings to expected values
-        const newSettings = {
-            ServiceSettings: {
-                EnableBotAccountCreation: true,
-            },
-        };
-        cy.apiUpdateConfig(newSettings);
+        cy.apiGetConfig().then(({config}) => {
+            origConfig = config;
+            const newConfig = {
+                ...origConfig,
+                ServiceSettings: {
+                    ...origConfig.ServiceSettings,
+                    EnableBotAccountCreation: true,
+                },
+            };
+
+            cy.apiUpdateConfig(newConfig);
+        });
```

Comment:
Please update to latest master for it has [`DeepPartial` applied to apiUpdateConfig](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/tests/support/api/system.d.ts#L100). Then revert the workaround wrapping with `cy.apiGetConfig()`.